fragmentCodeAndTransport.adoc

```diff
@@ -98,24 +98,29 @@ line 56. The least significant byte is output first, this means 32 is
 byte 0, 04 is byte 1 and and the final value 02 is byte 4.
 
 ==== Siemens transport
-
-The packet format is given in
-<<fig:packet-format>>. So this means the packet
-will be packed as follows:
+The packet is encapsulated according to the https://drive.google.com/file/d/1R-_koXIpdb9_qW6jpz74TSnNXOfJGhfn/view?usp=drive_link[Unformatted Trace & Diagnostic Data Packet Encapsulation for RISC-V Specification], with the following attributes: 
```

Comment:
I think that this should link to https://github.com/riscv-non-isa/e-trace-encap/releases/download/v1.0/e-trace-encap.pdf instead.  Someone at the Summit said that RVI is migrating away from Google because google.com is blocked in some countries.

There are additional places below that would also need to change.

Reply:
@rpsene @wmat can I get your input here?  I used the link from the Tech specs page (https://lf-riscv.atlassian.net/wiki/spaces/HOME/pages/16154769/RISC-V+Technical+Specifications), but what is the preferred way to refer to other specs?  The problem with using the link Paul suggested is that this will be wrong if/when we do an update for the E-Trace Encap spec.  I want something which always points to the latest approved/published version.

Reply:
I don't think that a drive.google.com link will always point to the latest, either.  It points to a specific file, similar to pointing to a file by inode rather than pointing to a file by name.

https://github.com/riscv-non-isa/e-trace-encap/releases/latest/ will always point to the latest release, though you have to do an additional click to get the PDF.

Reply:
@wmat, understood, and that’s why I picked the google drive link – I want the doc reference to always point to the latest published.  But with the move away from google docs, presumably the link for latest published is going to change to something else.  Do we know what this is yet?  Or are we going to have to scour all the specs for references to google links and replace them after that decision is made?

Reply:
If we go with the Google link then the downsides are:
- People in certain geographies (which constitute a significant percentage of RISC-V users) cannot follow the link today.
- If/when v1.0.1 of the encapsulation spec is released then the link will be obsolete (since it points directly to the v1.0 file).
- It will need to be updated when we migrate from Google (unless the Google link continues to work).

If we go with my original link then the downsides are:
- If/when v1.0.1 of the encapsulation spec is released then the link will be obsolete (since it points directly to the v1.0 file).
- We might want to update the link to point to something else when we migrate from Google (though I don't think that this would be mandatory until v1.0.1 comes out).

If we go with the "latest" link (in [my previous comment](https://github.com/riscv-non-isa/riscv-trace-spec/pull/165#discussion_r1817420560)) then the downsides are:
- People have to do an additional click to get the actual spec.

Or maybe we just don't have a link at all.  People would just have to go figure it out.

Reply:
Perhaps a link to "latest" with instructions in the GitHub README about what to do next?